pylint_odoo/checkers/no_modules.py

```diff
@@ -568,6 +576,28 @@ def visit_functiondef(self, node):
         if not node.is_method():
             return
 
+        if node.name.startswith('_compute_'):
```

Comment:
What about use `fields.*(compute='THIS_PARAMETER')`?
In order to search all valid computed fields with/without standard

Reply:
@moylop260 
I refactor my previous code to detect the `fields.*(compute='THIS_PARAMETER')`
I I added various code of example to test this new check